app/src/main/java/ca/sfu/pacmacro/PlayerActivity.java

```diff
@@ -249,9 +254,15 @@ AppCompatDialog createGpsAlertDialog() {
         AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(PlayerActivity.this, R.style.Theme_AppCompat_Dialog);
         dialogBuilder.setTitle(getApplicationContext().getString(R.string.warning));
         dialogBuilder.setMessage(getApplicationContext().getString(R.string.gps_alert_dialog_msg));
-        dialogBuilder.setCancelable(false);
+        dialogBuilder.setCancelable(true);
         dialogBuilder.setPositiveButton(getApplicationContext().getString(R.string.go_to_settings), null)
                 .create();
+        dialogBuilder.setNegativeButton("Do nothing",
```

Comment:
Can you change the text to "Go back" and force it to return to the _Select a Player_ screen?